### PR TITLE
Added type abbrev name to be used in media_settings.json for Dynamic transceiver tuning 

### PR DESCRIPTION
**Pull request by dgsudharsan**
Added short type to be used in media_settings.json for Dynamic transceiver tuning.

### CODE REVIEW

File: sonic_sfp/sff8024.py [hunk 1]
```diff
@@ -34,3 +34,33 @@
...
+    '0e': 'CXP',
+    '0f': 'HD4X',
+    '10': 'HD8X',
+    '11': 'QSFP28',
+    '12': 'CXP2',
+    '13': 'CDFP-1/2',
+    '14': 'HD4X-Fanout',
+    '15': 'HD8X-Fanout',
+    '16': 'CDFP-3',
+    '17': 'microQSFP',
```
Comment (lguohan) [inline sonic_sfp/sff8024.py]: MicroQSFP

File: sonic_sfp/sff8436.py [hunk 2]
```diff
@@ -352,6 +352,11 @@ class sff8436InterfaceId(sffbase):
                  'size':1,
                  'type' : 'enum',
                  'decode' : type_of_transceiver},
+             'short_type':
```
Comment (lguohan) [inline sonic_sfp/sff8436.py]: type_abbrv_name? @jleveque

File: sonic_sfp/inf8628.py [hunk 3]
```diff
@@ -6,7 +6,7 @@
 from __future__ import print_function
 
 try:
-    from .sff8024 import type_of_transceiver    # Dot module supports both Python 2 and Python 3 using explicit relative import methods
+    from .sff8024 import *    # Dot module supports both Python 2 and Python 3 using explicit relative import methods
```
Comment (stcheng) [inline sonic_sfp/inf8628.py]: could you avoid import star?

File: sonic_sfp/sff8024.py (hunk 1 above)
Comment (dgsudharsan) [reply sonic_sfp/sff8024.py]: Done

File: sonic_sfp/sff8436.py (hunk 2 above)
Comment (dgsudharsan) [reply sonic_sfp/sff8436.py]: Done

File: sonic_sfp/inf8628.py (hunk 3 above)
Comment (dgsudharsan) [reply sonic_sfp/inf8628.py]: Done

Comment (stcheng) [reply sonic_sfp/inf8628.py]: thanks

**Pull request merged**